vng_api_common/audittrails/viewsets.py

```diff
@@ -32,29 +32,35 @@ def create_audittrail(self, status_code, action, version_before_edit, version_af
         else:
             main_object = self.get_audittrail_main_object_url(data, self.audit.main_resource)
 
+        # Django 2.2 compatibility
+        if hasattr(self.request, 'headers'):
+            headers_attr = 'headers'
+        else:
+            headers_attr = 'META'
+
         applications = self.request.jwt_auth.applicaties
         if len(applications) > 1:
             logger.warning("Unexpectedly found %d applications, expected at most one", len(applications))
         if applications:
             application = applications[0]
             app_id, app_presentation = str(application.uuid), application.label
         else:
-            header = 'X_NLX_REQUEST_APPLICATION_ID'
-            if hasattr(self.request, 'headers'):
-                app_id = self.request.headers.get(header)
-            else:
-                app_id = self.request.META.get(header)
+            app_id = getattr(self.request, headers_attr).get('HTTP_X_NLX_REQUEST_APPLICATION_ID')
             app_presentation = app_id  # we don't have any extra information...
 
-        # Combine labels of all applicaties for the current client_id
-        applicatie_weergave = ', '.join(self.request.jwt_auth.applicaties.values_list('label', flat=True))
+        try:
+            user_id = self.request.jwt_auth.payload['user_id']
+        except KeyError:
```

Comment:
there's no fallback to NLX header if the value is falsy now - an empty string in JWT but a header that's present will give an incorrect result